packages/router/__tests__/router-test.ts

```diff
@@ -4577,7 +4568,7 @@ describe("a router", () => {
       });
     });
 
-    it("runs loaders above the boundary for 400 errors if binary data is attempted to be submitted using formMethod=GET", async () => {
+    it("runs loaders above the boundary for loader 400 errors", async () => {
```

Comment:
I think we can just delete this test entirely - we test boundary behavior for loader-thrown errors/responses elsewhere.  This test was specifically for the case where we threw from `normalizeNavigateOptions` before we ever ran any loaders - but that exception path is gone now

Reply:
Done